src/Deserializers/StatementListDeserializer.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Wikibase\DataModel\Deserializers;
+
+use Deserializers\Deserializer;
+use Deserializers\Exceptions\DeserializationException;
+use Wikibase\DataModel\Statement\StatementList;
+
+/**
+ * Package private
+ *
+ * @licence GNU GPL v2+
+ * @author Bene* < benestar.wikimedia@gmail.com >
+ */
+class StatementListDeserializer implements Deserializer {
+
+	/**
+	 * @var Deserializer
+	 */
+	private $statementDeserializer;
+
+	/**
+	 * @param Deserializer $statementDeserializer
+	 */
+	public function __construct( Deserializer $statementDeserializer ) {
+		$this->statementDeserializer = $statementDeserializer;
+	}
+
+	/**
+	 * @see Deserializer::deserialize
+	 *
+	 * @param mixed $serialization
+	 *
+	 * @return object
+	 * @throws DeserializationException
+	 */
+	public function deserialize( $serialization ) {
+		$this->assertHasGoodFormat( $serialization );
+
+		return $this->getDeserialized( $serialization );
+	}
+
+	private function getDeserialized( array $serialization ) {
+		$statementList = new StatementList();
+
+		foreach( $serialization as $statementArray ) {
```

Comment:
It indeed just reflects the current serialization format. I'm not sure if we should enforce statements grouped by property id in the serializatoin though if it is not done in the DataModel. This is a more general question but we should stay consistent and either do this grouping in Items/Properties as well or don't force it in the serialization.